### PR TITLE
phpstan static code analysis fixes

### PR DESCRIPTION
**Pull request by acrobat**
I am targeting this branch, because some code (style) fixes.

## Subject

Some code fixes discovered by phpstan's static code analysis

### CODE REVIEW

File: Block/Service/MenuBlockService.php [hunk 1]
```diff
@@ -83,7 +83,7 @@ public function execute(BlockContextInterface $blockContext, Response $response
             'context' => $blockContext,
         ];
 
-        if ('private' === $blockContext->getSettings('cache_policy')) {
+        if ('private' === $blockContext->getSetting('cache_policy')) {
```
Comment (greg0ire) [inline Block/Service/MenuBlockService.php]: lol nice

Comment (greg0ire) [reply Block/Service/MenuBlockService.php]: This fixes a bug, right? It's not just style...

Comment (acrobat) [reply Block/Service/MenuBlockService.php]: Yes this is indeed a bugfix!

**Comment by acrobat** [comment]
I will see if I can come up with a test for the bug that was fixed in this PR. Maybe move that bugfix + tests into a separate PR, what do you think @greg0ire? Just keep the pure code style changes here and move the bugfix to another PR

**Comment by greg0ire** [comment]
Either do that or do separate commits, I'm fine with both solutions.

**Comment by Soullivaneuh** [comment]
Thank you @acrobat

**Pull request merged**